stream/src/main/scala/eu/neverblink/jelly/pekko/stream/DecoderFlow.scala

```diff
@@ -0,0 +1,9 @@
+package eu.neverblink.jelly.pekko.stream
```

Comment:
sorry, one last thing – pls also rename the directory to `pekko-stream` to make it more consistent.

Also, in that case, should the module be named `jelly-pekko-stream`, without the `s` at the end? It would also be consistent with package and module naming in Pekko itself.